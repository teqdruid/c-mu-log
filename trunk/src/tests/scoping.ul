// test to demonstrate the local scoping of variables within a block

foo($x,?){
	$x>1;
	$x<10;
	}

bar($x,?){$x>0;
	$x<5;
	}

main(){
	foo($x,$z);
	bar($y,$d);
	@print($x,"and",$y,$z);
      }
