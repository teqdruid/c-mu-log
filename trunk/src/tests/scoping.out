1..10and0..5Any

 ^^^  Solution  ^^^ 

No more solutions
